dbScripts/makeAntInfoATRI.sql

```diff
@@ -25,46 +25,137 @@ CREATE TABLE ARA06(antId integer primary key, holeName text, antName text, polTy
 -- not clear which Hole corresponds to which FOAM. FOAM+T/B+H/V+daqChan OK
 -- which hole which trigChan?
 
-insert into ARA01 VALUES(0 , "BH1", "TVPol", "kVertical", 
-0, 1, 1, 1, 4, "kBicone",
--3.8242, 6.6507, -62.8374, 13.714,  
-0,0,0,0,  
+-- Pre 2018 channel mapping are exactly same  as post 2018 in A1 -- Mohammad
```

Comment:
Can you add this comment above the 2018 table below as well? Is there anything different about the tables? If not, let's make clear it's not just the channel mappings that are the same.

Reply:
The table values are exactly the same except for the table names. Pre-2018, the table has a name "ARA01" and post -2018, it's named "ARA01_2018". These entries from these names are called in geomtools based on years. So, keeping both is safe at the moment.

Reply:
Yes, agreed keeping both is no problem! I just mean let's make it clear that the tables are exactly the same, not just the channels mappings. Something like "Pre 2018 table is exactly the same as post 2018 table in A1."

Reply:
Thank you! Can you just add that comment above the post 2018 table as well? Just to make sure it's not missed